src/app/home.component.html

```diff
@@ -14,17 +14,21 @@
 <stache
   layout="container"
   showBreadcrumbs="false"
-  windowTitle="SKY UX 2"
+  windowTitle="SKY UX"
   pageTitle="Welcome to Blackbaud's user experience framework">
   <stache-page-summary>
       <p>SKY UX 2 is an updated version of Blackbaud's UX framework. The framework brings a consistent, cohesive experience to Blackbaud products and allows third-party developers to provide the same consistent experience in their customizations and applications. SKY UX 2 takes advantage of updates to <a href="https://angular.io/">Angular</a> to keep SKY UX modern while increasing the tooling, testing, and performance available.
       </p>
     </stache-page-summary>
 
-    <sky-alert alertType="info" [closeable]="false">
-      To follow along with the development of SKY UX and to make contributions, see <a href="https://github.com/blackbaud/skyux2">the SKY UX 2 GitHub repo</a>. The SKY UX 2 documentation lives alongside the components in the repo, so you can contribute to the documentation there as well. For information about how to contribute, see <a href="https://github.com/blackbaud/skyux2/blob/master/CONTRIBUTING.md">the SKY UX 2 contribution guidelines</a>.
+    <sky-alert alertType="warning" [closeable]="false">
+      This site details SKY UX support for the <a href="https://angular.io">latest version of Angular</a>. Visit our <strong><a href="https://skyux.developer.blackbaud.com">SKY UX 1 documentation</a></strong> for information related to <a href="https://angularjs.org">AngularJS (1.x)</a>.
     </sky-alert>
 
+    <p>
+      To follow along with the development of SKY UX and to make contributions, see <a href="https://github.com/blackbaud/skyux2">the SKY UX 2 GitHub repo</a>. The SKY UX 2 documentation lives alongside the components in the repo, so you can contribute to the documentation there as well. For information about how to contribute, see <a href="https://github.com/blackbaud/skyux2/blob/master/CONTRIBUTING.md">the SKY UX 2 contribution guidelines</a>.
```

Comment:
Shouldn't this be "the SKY UX GitHub repo"? Or is the 2 appropriate here?

Reply:
Yeah probably.  I changed none of that verbiage, just moved it out of an alert.  Honestly, I'd like to revamp this entire page, but at least wanted to get the warning message in there.